pkg/plugin/admissioncontrollers/webhook/admission_controller.go

```diff
@@ -294,6 +294,41 @@ func (c *admissionController) mutate(req *admissionv1.AdmissionRequest) *admissi
 	return admissionResponseBuilder(uid, true, "", patchBytes)
 }
 
+func (c *admissionController) processWorkload(req *admissionv1.AdmissionRequest) *admissionv1.AdmissionResponse {
+	var uid = string(req.UID)
+	requestKind := req.Kind.Kind
+
+	annotations, supported, err := c.annotationHandler.GetAnnotationsFromRequestKind(requestKind, req)
+	if !supported {
+		// Unknown request kind - pass
+		return admissionResponseBuilder(uid, true, "", nil)
```

Comment:
Is sending back nil error and true correct behaviour? I think GetAnnotationsFromRequestKind() method itself should return unsupported error message when corresponding kind is not supported

Reply:
Yes, workloads of unsupported type are let through. That is the original behavior.